examples/games/breakout.rs

```diff
@@ -249,11 +249,6 @@ fn setup(
     commands.spawn(WallBundle::new(WallLocation::Top));
 
     // Bricks
-    // Negative scales result in flipped sprites / meshes,
-    // which is definitely not what we want here
-    assert!(BRICK_SIZE.x > 0.0);
-    assert!(BRICK_SIZE.y > 0.0);
-
```

Comment:
why remove those?

Reply:
That's from https://rust-lang.github.io/rust-clippy/master/index.html#/assertions_on_constants

At a glance it didn't seem terribly useful anyhow (would someone really use negative values in a const called `BRICK_SIZE`?).

We could replace with `if ( ... ) { panic! }` or allow the lint or (something else?).

What do you think?

Reply:
I don't like them, but they were probably used for documentation and easy customisation of the example. @alice-i-cecile ?

Reply:
Maybe substitute for `assert!(n_columns > 0);` a bit lower? That produces a slightly less cryptic error when attempting to use a negative brick size.

Reply:
Or use `BRICK_SIZE.abs()` for calculations where this matters?

I guess this is for the scenario where the user has replaced the brick with a textured sprite and wants to flip it.

Reply:
My personal preference would be for `assert!(n_columns > 0);`. If a user has replaced the brick with a textured sprite, then `BRICK_SIZE` has now truly become a "scale" and not a "size". 

Maybe including a comment above `const BRICK_SIZE` pointing out that this **can't** be negative and pointing them to `flip_x` and `flip_y`.

Reply:
I would rather just remove the assertions than fuss around with the absolute value stuff here.